packages/terra-navigation-side-menu/src/_MenuItem.jsx

```diff
@@ -108,14 +110,16 @@ class MenuItem extends React.Component {
       text,
       ...customProps
     } = this.props;
+    const theme = this.context;
 
-    const itemClassNames = cx([
+    const itemClassNames = classNames(cx(
```

Comment:
nit: This formatting is a little awkward/inconsistent with the other PRs. Generally the cx( and ending params are new-lined